lib/globals.ts

```
@@ -0,0 +1,18 @@
+import { Json } from "@/database.types";
+import { merge } from "./merger";
+
+declare global {
+  interface ObjectConstructor {
+    merge<R extends Object | Json = Object, P extends Object | Json = Object>(
```

Biome

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)